apps/docs/src/app/platform/component-docs/platform-forms/platform-form-container/platform-form-container-examples/platform-field-layout/platform-field-isinline-change-example.component.html

```diff
@@ -0,0 +1,57 @@
+<fdp-form-group #fg1 [formGroup]="form" mainTitle="XL1-L1-M1-S1" columnLayout="XL1-L1-M1-S1">
+    <fdp-form-field
+        #ff1
+        id="basicTextarea"
+        label="Basic Textarea with Platform Forms"
+        placeholder="Start entering detailed description"
+        hint="This is tooltip help"
+        column="1"
+        rank="1"
+        hintPlacement="left"
+    >
+        <fdp-textarea name="basicTextarea" [formControl]="ff1.formControl"></fdp-textarea>
+    </fdp-form-field>
+    <fdp-form-field #ff2 id="seasons0" label="XL: true, L: false, M: true, S: false" column="1" rank="2">
+        <fdp-checkbox-group
+            [list]="seasons"
+            name="seasons0"
+            [inlineLayout]="{ XL: true, L: false, M: true, S: false }"
+            [isInline]="true"
+            ariaLabel="My favourite Seasons"
+            [formControl]="ff2.formControl"
+        ></fdp-checkbox-group>
+    </fdp-form-field>
+    <fdp-form-field
+        #ff3
+        id="basicTextarea2"
+        label="Basic Textarea with Platform Forms 2"
+        [placeholder]="'Start entering detailed description'"
+        hint="This is tooltip help"
+        column="1"
+        rank="3"
+    >
+        <fdp-textarea name="basicTextarea2" [formControl]="ff3.formControl"></fdp-textarea>
+    </fdp-form-field>
+    <fdp-form-field
+        #ff5
+        id="basicTextarea3"
+        label="Basic Textarea with Platform Forms 5"
+        [placeholder]="'Start entering detailed description'"
+        hint="This is tooltip help"
+        column="1"
+        rank="5"
+    >
+        <fdp-textarea name="basicTextarea3" [formControl]="ff5.formControl"></fdp-textarea>
+    </fdp-form-field>
+    <fdp-form-field
+        #ff6
+        id="basicTextarea6"
+        label="Basic Textarea with Platform Forms 6"
+        [placeholder]="'Start entering detailed description'"
```

Comment:
unnecessary binding, also in lines [28](https://github.com/SAP/fundamental-ngx/pull/6039/files#diff-36ec89c3d1f6108f42b5d02b21fae4926d731a851ce94cf0ceb16fd2754c8f21R28) and [39](https://github.com/SAP/fundamental-ngx/pull/6039/files#diff-36ec89c3d1f6108f42b5d02b21fae4926d731a851ce94cf0ceb16fd2754c8f21R39)

Reply:
fixed